src/java/org/apache/cassandra/schema/IndexMetadata.java

```diff
@@ -79,33 +82,65 @@ public enum Kind
     public final UUID id;
     public final String name;
     public final Kind kind;
+
+    /** Compression parameters to use for compressing primary keys of the table the index points to */
```

Comment:
Can we mention here that all indexes on one table share the same key compression?